lib/wickedbase.pm

```diff
@@ -148,6 +150,23 @@ sub get_current_ip {
     return;
 }
 
+=head2 download_all
+
+Download all files from data/wicked into WICKED_DATA_DIR.
+This method is used by before_test.pm.
+=cut
+sub download_all {
+    my $dst  = WICKED_DATA_DIR . '/data_wicked.tar.gz';
+    my $path = hashed_string($dst);
+    my $cmd  = "tar -czf $path -C '" . $bmwqemu::vars{CASEDIR} . "/data' wicked";
```

Comment:
so we downloading whole data directory , right ?

Reply:
no, only wicked.

Reply:
You could look into serial.log and search for `tar -zxvf` then you see which files are copied.

Reply:
decision after discussion: die if we try to get a file outside wicked directory. For those cases, it's better to use wget/curl directly instead of get_form_data function. If we need to download more and more files from other data directories, we can always add this logic later on.